src/bundle/Controller/VersionController.php

```diff
@@ -58,6 +58,7 @@ public function archiveActionsAction(
 
         if ($archiveActionsForm->isValid()) {
             $this->deleteVersionsBasedOnFormSubmit($archiveActionsForm, $content->id);
+            $this->createDraftVersionBasedOnFormSubmit($archiveActionsForm, $content->id);
```

Comment:
Given that you have a `Content`, is it really worth passing the ID ? It is gonna be loaded back to a `ContentInfo` later on. You might as well pass the `ContentInfo` directly.